sparkle/src/components/Dropdown.tsx

```diff
@@ -224,6 +224,7 @@ interface DropdownMenuContentProps
   mountPortal?: boolean;
   mountPortalContainer?: HTMLElement;
   dropdownHeaders?: React.ReactNode;
+  onOpenAutoFocus?: (e: React.FocusEvent<HTMLDivElement>) => void;
```

Comment:
Hmmm, where is it then used?

Reply:
I mean it's not used within DropdownMenuContent right? 🤔

Reply:
ha yes, it's not used by us, it's used by radix Content component ( passes in ...props )

Reply:
Oooh okay, I couldn't find where it was used 👌🏼